src/Extension/FluentExtension.php

```diff
@@ -968,16 +969,27 @@ protected function getRecordLocale()
 
     /**
      * Returns the source locale that will display the content for this record
+     * Source locale for frontend context is used by default as this is the most common use case,
+     * but you can optionally specify CMS context as well
+     * Passing null will fall back to whatever context is currently in use in global fluent state
+     *
+     * @param bool|null $isFrontend
      *
      * @return Locale|null
      */
-    public function getSourceLocale()
+    public function getSourceLocale(?bool $isFrontend = true): ?Locale
```

Comment:
```suggestion
    public function getSourceLocale()
```
Additional parameters (even optional ones) and explicit typehints cannot be added outside of a major release.
See [the definition of public API](https://docs.silverstripe.org/en/5/project_governance/public_api/)

Reply:
If those changes are required, they will need to target `8` so they can be included in the next major release.

Reply:
I'll see if I can find another way 🤔